android/src/main/java/io/parity/signer/domain/Models.kt

```diff
@@ -263,6 +263,9 @@ data class NetworkModel(
 	}
 }
 
+fun NetworkModel.createDefaultPath(): String = "//${this.title}"
```

Comment:
@krodak are you using the same logic on iOS?

Reply:
```swift
 let pathAndNetworks: [(path: String, network: MmNetwork)] = networks
            .map { (path: "//\($0.title)", network: $0) }
```

Reply:
`MNetworkDetails` model as a `path_id` and that's exactly the derivation path prefix.